src/styles/blocks/animations.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes move {
+  from {
+    transform: rotate(0);
```

Comment:
Add the unit 'deg' to the initial rotation value (0) for consistency: 'transform: rotate(0deg);'.